packages/react/src/table/TableScroller.tsx

```diff
@@ -311,6 +311,10 @@ export function TableScroller({ children }: TableScrollerProps) {
 							position: 'absolute',
 							top: 0,
 							touchAction: 'none', // Prevent default touch actions
+							// See https://www.w3.org/TR/CSS21/ui.html#system-colors
+							'@media (forced-colors: active)': {
```

Comment:
<img width="566" alt="image" src="https://github.com/user-attachments/assets/4ab4ffc2-ec35-4a9d-a309-491c1579f707">
And here's how native Windows scrollbars render 
<img width="298" alt="image" src="https://github.com/user-attachments/assets/5e27940a-7e1d-493c-b85a-3f4cdb73c6bc">